api/src/main/java/org/pmiops/workbench/db/dao/BillingProjectBufferEntryDao.java

```diff
@@ -22,18 +22,24 @@
 
   DbBillingProjectBufferEntry findByFireCloudProjectName(String fireCloudProjectName);
 
-  @Query("SELECT COUNT(*) FROM DbBillingProjectBufferEntry WHERE status IN (0, 2)")
-  Long getCurrentBufferSize();
+  @Query(
+      "SELECT COUNT(*) FROM DbBillingProjectBufferEntry WHERE status IN (0, 2) AND access_tier = (:tier)")
+  Long getCurrentBufferSizeForAccessTier(@Param("tier") String accessTier);
 
+  // TODO: decide if we care about potential "tier bias" here; it does not take tiers into account
```

Comment:
e.g. the potential of allocating N registered tier projects before starting the allocation of N controlled-tier.  In practice, I observed the BB Service to alternate.